### PR TITLE
Add document for aws endpoint setting

### PR DESCRIPTION
**Pull request by yuyuvn**
Add document for aws endpoint setting
https://github.com/kedacore/keda/pull/3779

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes https://github.com/kedacore/keda/issues/3337

### CODE REVIEW

File: content/docs/2.9/scalers/aws-dynamodb-streams.md [hunk 1]
```diff
@@ -27,6 +29,7 @@ triggers:
 **Parameter list:**
 
 - `awsRegion` - AWS Region for the DynamoDB.
+- `awsEndpoint` - An optional endpoint URL that overrides the default AWS endpoint.
```
Comment (JorTurFer) [inline content/docs/2.9/scalers/aws-dynamodb-streams.md:32]:
```suggestion
- `awsEndpoint` - Endpoint URL to override the default AWS endpoint. (Default: `""`, Optional)
```

File: content/docs/2.9/scalers/aws-dynamodb.md [hunk 2]
```diff
@@ -33,6 +35,7 @@ triggers:
 **Parameter list:**
 
 - `awsRegion` - AWS Region for the DynamoDB Table.
+- `awsEndpoint` - An optional endpoint URL that overrides the default AWS endpoint.
```
Comment (JorTurFer) [inline content/docs/2.9/scalers/aws-dynamodb.md:38]:
```suggestion
- `awsEndpoint` - Endpoint URL to override the default AWS endpoint. (Default: `""`, Optional)
```

File: content/docs/2.9/scalers/aws-kinesis.md [hunk 3]
```diff
@@ -29,6 +31,7 @@ triggers:
 - `shardCount` - The target value that a Kinesis data streams consumer can handle. (Default: `2`, Optional)
 - `activationShardCount` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional)
 - `awsRegion` - AWS Region for the Kinesis Stream.
+- `awsEndpoint` - An optional endpoint URL that overrides the default AWS endpoint.
```
Comment (JorTurFer) [inline content/docs/2.9/scalers/aws-kinesis.md:34]:
```suggestion
- `awsEndpoint` - Endpoint URL to override the default AWS endpoint. (Default: `""`, Optional)
```

File: content/docs/2.9/scalers/aws-sqs.md [hunk 4]
```diff
@@ -36,6 +38,7 @@ triggers:
 - `scaleOnInFlight` - Indication whether or not to scale on queued messages or to include in-flight messages as well.
   - When set to `false` "actual messages" is equal to `ApproximateNumberOfMessages`. When set to `true` "actual messages" is equal to `ApproximateNumberOfMessages` + `ApproximateNumberOfMessagesNotVisible`, since `NotVisible` in SQS terms means the message is still in-flight/processing. (default: true)
 - `awsRegion` - AWS Region for the SQS Queue.
+- `awsEndpoint` - An optional endpoint URL that overrides the default AWS endpoint.
```
Comment (JorTurFer) [inline content/docs/2.9/scalers/aws-sqs.md:41]:
```suggestion
- `awsEndpoint` - Endpoint URL to override the default AWS endpoint. (Default: `""`, Optional)
```

**Review by JorTurFer — COMMENTED** [review]
I suggested changes to follow the [style guide](https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler) for docs. There is also missing the change in cloudwatch scaler docs

File: content/docs/2.9/scalers/aws-cloudwatch.md [hunk 5]
```diff
@@ -46,6 +48,8 @@ triggers:
 
 **Parameter list:**
 
+- `awsRegion` - AWS Region for the AWS Cloudwatch.
```
Comment (yuyuvn) [inline content/docs/2.9/scalers/aws-cloudwatch.md:51]: awsRegion was missed

**Comment by yuyuvn** [comment]
Fixed style and add docs for cloudwatch
https://github.com/kedacore/keda-docs/pull/965/commits/ac588b486128456a510b381136eecb17548e289b

**Review by JorTurFer — APPROVED** [review]
LGTM!

**Pull request merged**